rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -513,6 +513,29 @@ void getPendingFederatorPublicKeyOfType() {
             assertThat(bridgeSupport.getPendingFederatorPublicKeyOfType(0, FederationMember.KeyType.MST), is(mstKey.getPubKey()));
         }
 
+        @Test
+        void getProposedFederationAddress_whenFederationSupportReturnsEmpty_shouldReturnEmpty() {
+            // Act
+            var actualProposedFederationAddress = federationSupport.getProposedFederationAddress();
+
+            // Assert
+            assertFalse(actualProposedFederationAddress.isPresent());
+        }
+
+        @Test
+        void getProposedFederationAddress_whenProposedFederationExists_shouldReturnAddress() {
+            // Arrange
+            var expectedAddress = federation.getAddress();
+            when(federationSupport.getProposedFederationAddress()).thenReturn(Optional.of(expectedAddress));
+
+            // Act
+            var actualProposedFederationAddress = federationSupport.getProposedFederationAddress();
```

Comment:
```suggestion
            var actualProposedFederationAddress = bridgeSupport.getProposedFederationAddress();
```

:)

Reply:
I agree. It seems here that we are testing the `FederationSupport`, but in the `BridgeSupportTest` instead of the `ProposedFederationTests`.

Reply:
Yes too much time working with FederationSupport, thanks :)

Reply:
jaja no worries